packages/service-integration/src/service_integration/oci_image_spec.py

```diff
@@ -0,0 +1,162 @@
+""" Support for Open Container Initiative (OCI)
+
+SEE https://opencontainers.org
+SEE https://github.com/opencontainers
+SEE https://github.com/opencontainers/image-spec/blob/main/annotations.md
+"""
+
+import os
+from datetime import datetime
+from typing import Any, Dict
+
+from models_library.basic_types import SHA1Str, VersionStr
+from pydantic import BaseModel, Field
+from pydantic.main import Extra
+from pydantic.networks import AnyUrl
+
+from .labels_annotations import from_labels, to_labels
+
+#
+# Prefix added to docker image labels using reverse DNS notations of a domain they own
+# SEE https://docs.docker.com/config/labels-custom-metadata/#key-format-recommendations
+#
+DOCKER_LABEL_PREFIXES = ("com.docker", "io.docker", "org.dockerproject")
+LS_LABEL_PREFIX = "org.label-schema"
+OCI_LABEL_PREFIX = "org.opencontainers.image"
+
+# See conversion rules https://github.com/opencontainers/image-spec/blob/main/annotations.md#back-compatibility-with-label-schema
+_TO_OCI = {
+    "build-date": "created",
+    "url": "url",
+    "vcs-url": "source",
+    "version": "version",
+    "vcs-ref": "revision",
+    "vendor": "vendor",
+    "name": "title",
+    "description": "descripton",
+    "usage": "documentation",
+}
```

Comment:
can we have a jsonschema for verification when running these services?
I will also need in particular the vcs-url in the dask use case.

Reply:
yes, models of OCI and LabelSchema are available and therefore jsonschemas of those